internal/kibana/ingestmanager/client.go

```diff
@@ -0,0 +1,16 @@
+package ingestmanager
+
+type Client struct {
```

Comment:
I used a dedicated client for Elasticsearch. Maybe there is something similar for Kibana?

Reply:
I have not found a reliable one so far. I suspect, over time, as Kibana APIs become more "public" we will see one emerge. At that point we can use it here.

Reply:
BTW, looks like there is some community interest in this too: https://github.com/elastic/go-elasticsearch/issues/182

Reply:
Oh, nice :)